fly/integration/error_handling_test.go

```diff
@@ -105,36 +108,38 @@ var _ = Describe("Fly CLI", func() {
+				exec.Command(flyPath, "-t", targetName, "hide-pipeline", "-p", "pipeline", "--team", nonExistentTeam)),
 			Entry("hijack command returns an error",
-				exec.Command(flyPath, "-t", targetName, "hijack", "--handle", "container-id", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "hijack", "--handle", "container-id", "--team", nonExistentTeam)),
 			Entry("jobs command returns an error",
-				exec.Command(flyPath, "-t", targetName, "jobs", "-p", "pipeline", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "jobs", "-p", "pipeline", "--team", nonExistentTeam)),
 			Entry("pause-job command returns an error",
-				exec.Command(flyPath, "-t", targetName, "pause-job", "-j", "pipeline/job", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "pause-job", "-j", "pipeline/job", "--team", nonExistentTeam)),
 			Entry("pause-pipeline command returns an error",
-				exec.Command(flyPath, "-t", targetName, "pause-pipeline", "-p", "pipeline", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "pause-pipeline", "-p", "pipeline", "--team", nonExistentTeam)),
 			Entry("unpause-job command returns an error",
-				exec.Command(flyPath, "-t", targetName, "unpause-job", "-j", "pipeline/job", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "unpause-job", "-j", "pipeline/job", "--team", nonExistentTeam)),
 			Entry("unpause-pipeline command returns an error",
-				exec.Command(flyPath, "-t", targetName, "unpause-pipeline", "-p", "pipeline", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "unpause-pipeline", "-p", "pipeline", "--team", nonExistentTeam)),
 			Entry("set-pipeline command returns an error",
-				exec.Command(flyPath, "-t", targetName, "set-pipeline", "-p", "pipeline", "-c", "fixtures/testConfig.yml", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "set-pipeline", "-p", "pipeline", "-c", "fixtures/testConfig.yml", "--team", nonExistentTeam)),
 			Entry("destroy-pipeline command returns an error",
-				exec.Command(flyPath, "-t", targetName, "destroy-pipeline", "-p", "pipeline", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "destroy-pipeline", "-p", "pipeline", "--team", nonExistentTeam)),
 			Entry("get-pipeline command returns an error",
-				exec.Command(flyPath, "-t", targetName, "get-pipeline", "-p", "pipeline", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "get-pipeline", "-p", "pipeline", "--team", nonExistentTeam)),
 			Entry("order-pipelines command returns an error",
-				exec.Command(flyPath, "-t", targetName, "order-pipelines", "-p", "pipeline", "--team", "doesnotexist")),
+				exec.Command(flyPath, "-t", targetName, "order-pipelines", "-p", "pipeline", "--team", nonExistentTeam)),
 		)
 
 		DescribeTable("and you are NOT authorized to view the team",
```

Comment:
Hey @techgaun , btw thanks for you PR! :) Could you add a test for this case, when user is not authorized to view the team?

Reply:
@izabelacg my bad, I thought I added that one too. Updated now